feed.go

```diff
@@ -37,6 +41,21 @@ type Item struct {
 	Content     string
 }
 
+// SerializeOpts represents serialization options for a feed.
+type SerializeOpts struct {
+	Indent int // how many spaces to indent
+}
+
+// get the indentation string for this option set
+func (s SerializeOpts) indentString() string {
+	return strings.Repeat(" ", s.Indent)
+}
+
+// DefaultOptions are the default serialization options for the module. This
+// object can be mutated to update the default options for the module.
+var DefaultOptions = &SerializeOpts{Indent: defaultIndent}
```

Comment:
I don't think this should be exported. You can run into the issue where multiple packages are using the library and each of them wants to set their own default options. Best to just keep it private, if someone wants to change the options they can do it the manual way.